src/db/notifications.ts

```diff
@@ -0,0 +1,12 @@
+import { Collection, ObjectId } from 'mongodb';
+import { getCollection } from './mongo';
+
+export interface NotificationDoc {
+    _id: ObjectId;
+    region: string;
+    unsubscribeList: Array<string>;
+    subscribeList: Array<string>;
+}
+
+export default (): Collection<NotificationDoc> =>
+    getCollection<NotificationDoc>('Notifications');
```

Comment:
I didn't catch this last time, but the typical convention is to make collections lower case and plural (as you already have).